model/trace/database.yaml

```diff
@@ -21,17 +37,19 @@ groups:
         requirement_level: opt_in
 
   - id: trace.db.common.query_and_collection
-    extends: attributes.db.client.minimal
+    extends: trace.db.common.minimal
     type: attribute_group
     brief: This group defines the attributes used to perform database client calls.
     attributes:
       - ref: db.query.text
+        sampling_relevant: true
         requirement_level:
           recommended: >
             SHOULD be collected by default only if there is sanitization that excludes sensitive information.
       - ref: db.query.parameter
```

Comment:
Was there a specific reason to not include this? If we have `db.query.text` marked as sampling relevant and users opt in to collect query parameters, then this will be naturally also captured during spar start.

Definitely not pushing to add this as sampling relevant - just raising for consideration.

Reply:
from discussion:
* sampling relevant attributes incur some performance because can't defer setting them until after sampling decision
* sampling relevant attributes are somewhat related to metrics (which need to be captured on start)
* we think we can change an attribute from "not sampling relevant" to "sampling relevant" in the future (but not vice versa)